packages/html/src/timedurationpicker/tests/timedurationpicker.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { TimePicker } from '../../timepicker';
+
+const root = ReactDOM.createRoot(
+    document.getElementById('app') as HTMLElement
+);
+
+const styles = `
+    #test-area {
+        max-width: 660px;
+    }
+`;
+
+root.render(
+    <>
+        <style>{styles}</style>
+        <div id="test-area" className="k-d-grid k-grid-cols-2">
+
+            <span>TimeDurationPicker</span>
+            <span>TimeDurationPicker RTL</span>
+
+            <div>
+                <TimePicker placeholder="02 days : 01 hours : 30 mins" />
```

Comment:
We should use `<TimeDurationPicker />` here. This is applicable for all test cases in this test file.